proxy/src/auth/credentials.rs

```diff
@@ -61,7 +61,15 @@ impl<'a> ClientCredentials<'a> {
         // Project name might be passed via PG's command-line options.
         let project_option = params
             .options_raw()
-            .and_then(|mut options| options.find_map(|opt| opt.strip_prefix("project=")))
+            .and_then(|mut options| {
+                options.find_map(|opt| {
+                    if opt.starts_with("project=") {
+                        opt.strip_prefix("project=")
+                    } else {
+                        opt.strip_prefix("endpoint=")
+                    }
+                })
+            })
```

Comment:
What will happen if someone passes both of these parameters in the options?

Reply:
> Do you think we need to abrupt the connection if two options exist?

I think it should use `endpoint` parameter if it exists, if not, then check `project` as a fallback.
Now it looks like some undefined behaviour, I'd like to have it consistent (an option with abrupting connection suits too)